gensim/models/coherencemodel.py

```diff
@@ -91,7 +101,7 @@ def __init__(self, model=None, topics=None, texts=None, corpus=None, dictionary=
         else:
             self.dictionary = dictionary
```

Comment:
`None` doesn't work here actually. When no word->id mapping is provided while creating an LdaModel [this](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/ldamodel.py#L279) line is called which returns a `FakeDict` from [here](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/utils.py#L558). So this code:
```python
tm1 = LdaModel(corpus=corpus, num_topics=2, iterations=0, passes=0, random_state=10)
if tm1.id2word is None:
    print 'aye'
else:
    print 'naye'
```
actually prints `naye` but if I change it to `isinstance(tm1.id2word, FakeDict)` it outputs correctly.
Am I correct here?